src/Topshelf/Runtime/Windows/WindowsServiceHost.cs

```diff
@@ -268,6 +296,10 @@ protected override void Dispose(bool disposing)
 
         void CatchUnhandledException(object sender, UnhandledExceptionEventArgs e)
         {
+            // Call the custom exception handler if it is not null
+            //

```

Comment:
Yeah, a bunch of places you have a stray empty comment line; go ahead and remove it.